src/main/java/org/konkuk/klab/mtot/service/FriendshipService.java

```diff
@@ -22,10 +23,10 @@ public class FriendshipService {
     private final MemberRepository memberRepository;
 
     @Transactional
+    // 친구 요청 보내기 (친구 요청 DB에 저장)
     public FriendshipResponse requestFriend(FriendshipRequest request){
-        List<Member> memberList = findMemberIds(request.getRequesterEmail(), request.getReceiverEmail());
-        Member requester = memberList.get(0);
-        Member receiver = memberList.get(1);
+        Member requester = findMemberId(request.getRequesterEmail());
+        Member receiver = findMemberId(request.getReceiverEmail());
```

Comment:
`Optional`의 함수들을 사용합시다!
지금 `develop` 브랜치랑 충돌이 안 나니까 그쪽거를 이쪽으로 머지해와서 Exception으로 하는 게 좋을 것 같네요
아래는 없으면 Throw하고, 있으면 가져오게끔 진행합니다. 편하죠~
```suggestion
        Member requester = memberRepository.findByEmail(request.getRequesterEmail())
                .orElseThrow(MemberNotFoundException::new);
        Member receiver = memberRepository.findByEmail(request.getReceiverEmail())
                .orElseThrow(MemberNotFoundException::new);
```